File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        long long  sum1 = 0;
        for(int num : nums) sum1 += num;
        long long sum2 = (n * (n+1)) / 2;
        return abs(sum1 - sum2);
        
    }
};

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        long sum1 = (n * (n+1)) / 2;
        long sum2 = 0;
        for(int num : nums) sum2 += num;

        return (int) Math.abs(sum1 - sum2);

    }
}